.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+// https://prettier.io/docs/en/integrating-with-linters.html
+
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: ['plugin:prettier/recommended'],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
```

Comment:
I'm confused how es7 isn't supported if we're setting this. Should we be using `babel-eslint` or something? This is the type of thing I never remember and just have to research, but ideally we can lint all the files if possible. Mind investigating?

Reply:
Yeah I misspoke calling it ES7, it's still just a proposal (at stage 3): https://github.com/tc39/proposal-static-class-features

Reply:
Personally I don't see much benefit in adding a plugin so we can use an experimental syntax, when the lack of it isn't that bad (we already do a lot of post-facto statics like `Foo.propTypes = { ... }` a lot anyway for functional components). Curious what others think.

Reply:
ah ok, proposal gotcha ... maybe clarify the corresponding comment then